### PR TITLE
Fix exclusions for gaia and terraforming building

### PR DESCRIPTION
**Pull request by Grummel7**

### CODE REVIEW

**Comment by geoffthemedio** [comment]
This breaks the Gaia Transformation pedia article for me. Not sure if it's an issue with the script or some weird interaction...

**Comment by Grummel7** [comment]
> This breaks the Gaia Transformation pedia article for me. Not sure if it's an issue with the script or some weird interaction...

Looks okay for me:
gaia_transformation

Well, the scroll bar doesn't fit, but that seems to be a general problem, possibly related to #3763

**Comment by geoffthemedio** [comment]
Sorry, it was actually just broken if the UI option show autogenerated effects descriptions is on, and master also doesn't work.

File: default/scripting/buildings/TERRAFORM.focs.txt [hunk 1]
```diff
@@ -111,15 +115,15 @@ BuildingType
...
         OwnedBy empire = Source.Owner
         TargetPopulation low = 1
         Not Planet type = [Asteroids GasGiant]
         Not Planet environment = [Uninhabitable Good]
         Not Contains Building name = "BLD_GAIA_TRANS"
+        Not Enqueued type = Building name = "BLD_GAIA_TRANS"
         Not HasSpecial name = "GAIA_SPECIAL"
-[[FOR_ALL_TERRAFORM_PLANET_TYPES(        Not Contains Building name = "BLD_TERRAFORM_,")]]
-[[FOR_ALL_TERRAFORM_PLANET_TYPES(        Not Enqueued type = Building name = "BLD_TERRAFORM_,")]]
+        [[FOR_ALL_TERRAFORM_PLANET_TYPES(Not Contains Building name = "BLD_TERRAFORM_,")]]
```
Comment (geoffthemedio) [inline default/scripting/buildings/TERRAFORM.focs.txt:126]: could you add a comment with an example of what this produces? mentally parsing the script bits within the macro bits is difficult.

Comment (geoffthemedio) [reply default/scripting/buildings/TERRAFORM.focs.txt:126]: I meant specifically this macro

Comment (Grummel7) [reply default/scripting/buildings/TERRAFORM.focs.txt:126]: I didn't even write this, just reformatted it and copied it to Gaia Transformation.
Anyway, I've added a comments.
I also noticed that Gaia Transformation does not for a complex enqueue_location, so I simplified it.

**Pull request merged**